api/v1/routes/help_center.py

```diff
@@ -0,0 +1,47 @@
+from fastapi import APIRouter, Query, HTTPException
+from fastapi import Depends
+
+from sqlalchemy.orm import Session
+
+from api.core import responses
+from api.db.database import get_db
+from api.v1.models.article import Article
+from api.v1.schemas.article import ArticleResponse, SearchResponse
+
+from fastapi_limiter.depends import RateLimiter
+
+app = APIRouter()
+
```

Comment:
Don't name your router `app`, The FastApi instance is called `app` --- This is bad